StyleCop.Analyzers/StyleCop.Analyzers/SpacingRules/SA1013ClosingCurlyBracketsMustBeSpacedCorrectly.cs

```diff
@@ -60,12 +60,12 @@ private void HandleSyntaxTree(SyntaxTreeAnalysisContext context)
             {
                 switch (token.Kind())
                 {
-                case SyntaxKind.CloseBraceToken:
-                    this.HandleCloseBraceToken(context, token);
-                    break;
+                    case SyntaxKind.CloseBraceToken:
+                        this.HandleCloseBraceToken(context, token);
+                        break;
 
-                default:
-                    break;
+                    default:
+                        break;
                 }
```

Comment:
:bulb:  replace `switch` with
    if (token.IsKind(SyntaxKind.CloseBraceToken))
    {
        HandleCloseBraceToken(context, token); 
    }

and make `HandleCloseBraceToken` static